locales/en.json

```diff
@@ -219,7 +219,7 @@
     "user_info_failed" : "Unable to retrieve user information",
     "user_home_creation_failed" : "Unable to create user home folder",
 
-    "package_unexpected_error" : "An unexpected error occured with the package '{pkgname}'",
+    "package_unexpected_error" : "An unexpected error occured in the package '{pkgname}'",
```

Comment:
Ok. Good to know @jeromelebleu. I may know English better, but you guys understand the technical reality the error message represents better than I in many cases. In this case it sounds like both "in" and "with" are roughly equally (in)correct. The really accurate way of stating should actually be more along the lines of:
"An unexpected error occurred processing the package '{pkgname}'"